src/statusBar.ts

```diff
@@ -36,7 +46,15 @@ export class StatusBar {
 
 		this.statusBarItem.name = "Biome";
 		this.statusBarItem.command = Commands.ServerStatus;
+
+		this.statusBarUpdateItem = window.createStatusBarItem(
+			"biome.update",
+			StatusBarAlignment.Right,
+			-2,
```

Comment:
What does `-2` mean in this context?

Reply:
This value pushes the status bar item all the way to the right, or at least after Biome's main status bar item.

It the "priority"